scout/build/disease.py

```diff
@@ -70,8 +70,11 @@ def build_disease_term(disease_info, alias_genes={}):
                 for hgnc_id in alias_genes[hgnc_symbol]["ids"]:
                     hgnc_ids.add(hgnc_id)
         else:
-            LOG.debug("Gene symbol %s could not be found in database", hgnc_symbol)
+            hgnc_symbols_not_found.add(hgnc_symbol)
 
+    LOG.debug(
```

Comment:
If you add `if hgnc_symbols_not_found:` it becomes more complex than 15?

Reply:
naaaah, lets.